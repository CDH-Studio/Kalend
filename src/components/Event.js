import React from "react";
import { Text, View } from "react-native";

class Greeting extends React.Component {
  render() {
    return (
      <View>
        <Text>Hi, {this.props.name}!</Text>
      </View>
    );
  }
}

class Event extends React.Component {
  render() {
    return (
      <View>
        <Greeting name="Marie" />
      </View>
    );
  }
}

export default Event;
