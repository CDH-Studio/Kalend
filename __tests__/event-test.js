import React from 'react';
import Event from '../src/components/Event';
import renderer from 'react-test-renderer';

test('Event renders correctly', () => {
    const tree = renderer.create(<Event/>).toJSON();
    expect(tree).toMatchSnapshot();
  });