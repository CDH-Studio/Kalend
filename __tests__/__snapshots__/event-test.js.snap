// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event renders correctly 1`] = `
<View>
  <View>
    <Text>
      Hi, 
      Marie
      !
    </Text>
  </View>
</View>
`;
